### ISSUE TITLE
Log likelihood in the pda.define.llik.fn function

### ISSUE THREAD

**Issue by para2x** [PecanProject/pecan#2809]
### Bug Description

@mdietze It appears to me that there is a typo in how the Log-likelihood  is defined here :

https://github.com/PecanProject/pecan/blob/ee38544e1c995f47073019bd31b3179e7e64991a/modules/assim.batch/R/pda.define.llik.R#L30

This may need to be changed to :

```
LL <- (llik.par$n/2) * log(llik.par$par) -   (pda.errors/(llik.par$par*2))
```

Can you confirm that, Or I'm making a mistake somewhere in my math ?

**Comment by mdietze** [comment]
@istfer can chime in, but @para2x I think you wrote your LL down in terms of sigma (variance) while it appears Istem has it implemented as a precision. As far as I can tell, her math checks out. Precisions are faster and more precise because there's no need for inverses

**Comment by para2x** [comment]
Yes, That makes sense, I assumed `llik.par` is sigma, but if its the precision I think it checks out.

**Comment by istfer** [comment]
Yes it's precision, hyperpriors are set up that way

Although there is a bug in laplace likelihood in the pda.calc.error function, it's on my to-do list to make a PR for that, mentioning just in case you are currently using it
